src/main/java/se/kth/sda6/skeleton/posts/PostService.java

```diff
@@ -2,33 +2,39 @@
 
 import java.util.List;
 import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 /*
     @TODO Autowire the PostRepository and use it to implement all the service methods.
  */
+@Service
 public class PostService {
+    @Autowired
+    private PostRepository repository;
```

Comment:
repository would be more explicit if it was called postRepository (such as in CommentService, the name is commentRepository)